_layouts/impactform.html

```diff
@@ -1,3 +1,5 @@
+{% assign l = site.data.l10n.[page.lang] %}
```

Comment:
as the "l" variable is not used further down in this form, this can be removed as well.